packages/manifold/src/Manifold.ts

```diff
@@ -202,12 +202,9 @@ export const getFees = async (
         (isExclusive ? parseEther('0.00069') : parseEther('0.0005')) *
         quantityToMint
 
-      // calculate action fee
-      const mintPrice = data.mintPrice
-      let actionFee = 0n
-      if (mintPrice && typeof mintPrice === 'number') {
-        actionFee = parseEther(mintPrice.toString()) * quantityToMint
-      }
+      const { value, currency } = data.publicData.mintPrice
```

Comment:
`mintPrice` doesn't appear to be nested in publicData in their API docs 

https://docs.manifold.xyz/v/manifold-for-developers/tools-and-apis/claims-apis/get-claim-by-instance-id#get-claim-data-from-an-instance-id

Reply:
Here is a couple sample api calls

https://apps.api.manifoldxyz.dev/public/instance/data?id=46385392
https://apps.api.manifoldxyz.dev/public/instance/data?id=536662256